README.md

```diff
@@ -66,10 +67,35 @@ Continuous Deployment by [Visual Studio App Center](https://appcenter.ms) for Na
  <img src="https://user-images.githubusercontent.com/17120764/46603258-c639ce00-cb1c-11e8-9e43-b10c1a9b120b.png" width=300 />
 
 ### iOS
- 
-In theory, it should be can be run on iOS simulator using `yarn ios` command. Have no Mac machine to test it. 😅
- 
- 
+
+You have to run `yarn` or `npm install` first to download the dependencies.
+Then simply run `yarn ios`.
+
+If you got an error related to `AppCenter`, you should follow the steps below,
+These are some steps to install AppCenter on Mac OS:
+
+1. make sure you have installed cocoapods, if you haven't, then run this command on your terminal
+
+``` $ sudo gem install cocoapods --source http://rubygems.org ```
+2. install app center by executing this command :
+
+``` $ yarn add appcenter appcenter-analytics appcenter-crashes --exact ```
+3. Link the plugins to the React Native app by using the react-native link command.
+```
```

Comment:
![image](https://user-images.githubusercontent.com/26213148/47690439-9912a400-dc20-11e8-82f7-a633ea53fb20.png)


a bit messy, sir.
let me update it first.. :D

Reply:
@andangrd yeah, seems `react-native link` doesn't care whether the package already exists or not.